google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmCloudConfigurable.java

```diff
@@ -98,19 +143,25 @@ public JComponent createComponent() {
 
   @Override
   public boolean isModified() {
-    return !Comparing.strEqual(getCloudSdkLocation(), configuration.getCloudSdkPath()) ||
-        !Comparing.strEqual(getCloudProjectName(), configuration.getCloudProjectName());
+    boolean isSdkDirModified = !Comparing.strEqual(getCloudSdkDirectory(),
+        CloudSdkUtil.toParentDirectory(configuration.getCloudSdkExecutablePath()));
```

Comment:
I tend to prefer aligning parameters of the method if broken into multiple lines.